dotnet/src/MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders/StringDictionarySchemaObjectBuilder.cs

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2022-2023 Open Text.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System.Collections.Generic;
+
+namespace MicroFocus.FAS.AdapterSdkSchema.SchemaObjectBuilders
+{
+    public static class StringDictionarySchemaObjectBuilder
+    {
+        public static ISchemaObjectBuilder Create(Dictionary<string, IList<string>> metadata, IJsonStringBuilder jsonStringBuilder)
```

Comment:
We should be using `IDictionary<>` rather than `Dictionary<>`.  The value type is more debateable but I'd lean towards `IEnumerable<>` rather than `IList<>`.